superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -171,8 +200,24 @@ export default class ResultSet extends React.PureComponent<
       appContainer?.getAttribute('data-bootstrap') || '{}',
     );
 
-    const datasets = await getByUser(bootstrapData.user.userId);
-    const userDatasetsOwned = datasets.map(
+    const queryParams = rison.encode({
+      filters: [
+        {
+          col: 'owners',
+          opr: 'rel_m_m',
+          value: bootstrapData.user.userId,
+        },
+      ],
+      order_column: 'changed_on_delta_humanized',
+      order_direction: 'desc',
+    });
+
+    const response = await makeApi({
+      method: 'GET',
+      endpoint: `/api/v1/dataset?q=${queryParams}`,
+    })({});
```

Comment:
`makeApi` supports [`requestType: 'rison'`](https://github.com/airbnb/incubator-superset/blob/3fb4e5469041b439bd2bc036b8e060e8187025b5/superset-frontend/src/views/CRUD/alert/AlertList.tsx#L62). You don't have to encode the query params yourself.